lib/shiny-events.js

```diff
@@ -594,6 +594,10 @@ function shouldIgnore(message) {
     return true;
   }
 
+  if (/h/.test(message)) {
```

Comment:
Can we make this a more exact test? :)

Reply:
doh, I meant to fix that after I got things working. Thanks!